### ISSUE TITLE
Expansion to PlaceholderAPI

### ISSUE THREAD

**Issue by CDWimmer** [Multiverse/Multiverse-Core#2686]
### Feature request

**Feature description**
tl;dr: An expansion to PAPI support to get the formatted name of *any* world, not just the world you're in.

In addition to the `%multiverse_world_name_colored%` placeholder, that just grabs the MV formatted name of the world that the contextual player is in, I'd like to suggest a similar placeholder such that you can insert the name of the world you want the name of. 
This would look something like `%multiverse_world_name_colored_<world_name>%` which would find the world with internal name inserted in place of <world_name> and fetch that with its MV formatting/alias. 

**How the feature is useful**
I came across this need while using Denizen to generate a UI for teleportation and wanted some item lore to contain world names, coloured how it is in the chat prefix. Denizen however has no way to get the alias of a world out of multiverse, unless I went to the trouble of totally replacing MV with denizen and I really don't want to do that. 
Likewise, other plugins that generate UIs and similar where one might want to have the name of a world shown would benefit from the ability to display the pretty version of those world names, rather than the raw directory names ("The Nether" in red l looks nicer than "survival_nether"!)

**Comment by nicegamer7** [comment]
I will try to add this in the next couple days.

**Comment by benwoo1110** [comment]
Wait @nicegamer7 where is the code for mv’a papi?

**Comment by nicegamer7** [comment]
I asked the PAPI people on Discord to make the code available some time ago so that if we ever needed to add anything we could, and they did. It's in their repo, https://github.com/PlaceholderAPI/Multiverse-Expansion.

**Comment by CDWimmer** [comment]
> I will try to add this in the next couple days.

Thanks! will be a nice addition. Wasn't sure if I should ask here or over on the PAPI repo

**Comment by CDWimmer** [comment]
Any update? Sorry to nag

**Comment by nicegamer7** [comment]
Just opened a PR.

**Comment by benwoo1110** [comment]
Tbh I feel like this such just be done for all placeholders lol, if it end/starts with a worldname use that instead of the player’s world location.

**Comment by nicegamer7** [comment]
Yeah that's a nice idea. I'll see to it tomorrow.

**Comment by zax71** [comment]
if you need this now there is an unofical PlaceholderAPI Ecloud module [with the wiki page here](https://github.com/Multiverse/Multiverse-Core/wiki/Placeholders)

**Comment by CDWimmer** [comment]
> if you need this now there is an unofical PlaceholderAPI Ecloud module [with the wiki page here](https://github.com/Multiverse/Multiverse-Core/wiki/Placeholders)

Isn't that the thing nicegamer7 made a PR for? https://github.com/PlaceholderAPI/Multiverse-Expansion

**Comment by zax71** [comment]
Implemented in #2888 in the MV5 branch
